.gitignore

```diff
@@ -1,30 +1,31 @@
+# coverage reports
 */coverage.out
 extensions/*/coverage.out
+
+# unit test reports
 */checkstyle.xml
 extensions/*/checkstyle.xml
+
+# nodejs / yarn
 */package-lock.json
+node_modules/
+yarn-error.log
+yarn.lock
 
-ocis/config/identifier-registration.yaml
+# build artifacts
 */bin
 extensions/*/bin
 dist/
-/hugo
-*.key
-*crt
-node_modules/
-*/assets
 extensions/*/assets
 ocis/ocis
 ocis/cmd/ocis/__debug_bin
 ocis/cmd/ocis/config/
 
-.idea
-
-yarn-error.log
-yarn.lock
+# docs
+/hugo
 
-# Konnectd
-idp/assets/identifier
+# IDEs
+.idea
```

Comment:
Note: the things to be ignored for an IDE depends on the IDE that someone uses. Developers probably should create for themselves a global `.gitignore` file on their local system that mentions the things that their IDE(s) create. That avoids having to try and cover all IDEs in the `.gitignore` of every repo.

For example, see the post: https://david-brown.dev/posts/global-gitignore/

But it doesn't hurt to have this very common one in here.